piker/brokers/ib.py

```diff
@@ -1295,10 +1297,14 @@ def pack_position(pos: Position) -> dict[str, Any]:
     else:
         symbol = con.symbol
 
+    symkey = '.'.join([
+        symbol.lower(),
+        (con.primaryExchange or con.exchange).lower(),
```

Comment:
This fixes a bug where on pp loading from the broker, a sub-string of the actual intended instrument symbol would match showing a pp indicator for the wrong chart. explicitly truncating and appending the exchange solves this.